root_github.tf

```diff
@@ -0,0 +1,102 @@
+module "github_oidc_provider" {
+  source      = "./tdr-terraform-modules/identity_provider"
+  audience    = "sts.amazonaws.com"
+  thumbprint  = "6938fd4d98bab03faadb97b34396831e3780aea1"
```

Comment:
Assuming this is OK to be exposed in the repo? Would this be better in the tdr-configurations repo?

Reply:
It's a public thumbprint GitHub provide. I got it from here https://github.blog/changelog/2022-01-13-github-actions-update-on-oidc-based-deployments-to-aws/ so I think it's ok.

Reply:
Fair enough.